internal/statemachine/classic_test.go

```diff
@@ -612,25 +612,23 @@ func TestCustomizeCloudInit(t *testing.T) {
 		{
 			MetaData:      "foo: bar",
 			NetworkConfig: "foobar: foobar",
-			UserData: &[]imagedefinition.UserData{
-				{UserName: "ubuntu", UserPassword: "ubuntu"},
-				{UserName: "john", UserPassword: "password"},
-			},
+			UserData:      "bar: baz",
 		},
 		{
 			MetaData:      "foo: bar",
 			NetworkConfig: "foobar: foobar",
-			UserData:      nil,
+			UserData:      "",
 		},
 		{
 			NetworkConfig: "foobar: foobar",
-			UserData:      &[]imagedefinition.UserData{},
+			UserData:      "",
 		},
 		{
-			UserData: &[]imagedefinition.UserData{
-				{UserName: "ubuntu", UserPassword: "ubuntu"},
-				{UserName: "john", UserPassword: "password"},
-			},
+			UserData: `chpasswd:
+  expire: true
+  list:
+    - ubuntu:ubuntu
```

Comment:
Unfortunately this syntax is [now deprecated](https://cloudinit.readthedocs.io/en/latest/topics/modules.html#set-passwords) (and whinges about it in the [kinetic dailies](https://bugs.launchpad.net/ubuntu/+source/cloud-init/+bug/1991261/comments/12)). This is just a test, but it may be worth including the non-deprecated syntax too (or instead) just in case anyone reuses this stuff in other tests.